src/styles.scss

```diff
@@ -7,12 +7,40 @@
   font-family: 'Open Sans', sans-serif;
 }
 
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url("assets/fonts/avenir-heavy.ttf") format("truetype");
+}
+@font-face {
+  font-family: Avenir-Med;
+  src: url("assets/fonts/avenir-med.ttf") format("truetype");
+}
+
 html, body {
   height: 100%;
 }
 body {
     margin: 0;
     padding: 0;
+    background-color: #ffffff;
+}
+h1 {
```

Comment:
This changes all of the header tags in the entire website. Remove from "styles.scss" and put into "splashpage.component.scss." Also, it is better practice to use classes or nesting than styling all tags.